frontend/packages/client/src/components/Proposal/VoteOptions/VoteOptions.js

```diff
@@ -45,9 +46,9 @@ const VoteOptions = ({
 
   const canVote = isActive && (hasntVoted || !loggedIn);
 
-  const voteClasses = `vote-options border-light rounded mb-6${
-    !hasntVoted ? ' is-voted' : ''
-  }`;
+  const voteClasses = 'vote-options border-light rounded mb-6$';
+  const disabledClass = !canVote ? 'is-disabled' : '';
+  const votedClass = !hasntVoted ? 'is-voted' : '';
```

Comment:
and here just remove line 51, so when proposal is closed, upcoming or user voted the option has no opacity